narayana-release-process.sh

```diff
@@ -249,4 +182,4 @@ then
 fi
 
 # not sure why we need to look at CI at this point so commenting it out
-# xdg-open http://narayanaci1.eng.hst.ams2.redhat.com/ &
+# xdg-open https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/ &
```

Comment:
Could I remove this comment?

Reply:
I think the specific locations can be removed but to the idea of checking CI I think might have been to look at the post release jobs to see if they showed anything unexpected (like Maven artifacts unavailable or something like that?)

Reply:
The three comments themselves in this block could be changed to:

# Check that the quickstarts work with the artifacts deployed to Maven central and obtain the performance numbers for the release for publishing

Reply:
That could go into https://github.com/jbosstm/narayana/wiki/Narayana-Release-Process

Reply:
Sure, I will open a separate PR for this. Thanks